src/components/atoms/ProductImg.tsx

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+
+interface ProductImg {
+  size: string;
+  src: string;
+}
+
+const ProductImg = ({ size, src }: ProductImg) => {
+  return (
+    <ImgContainer style={{ width: size, height: size }}>
+      <Image src={src || ''} />
+    </ImgContainer>
+  );
+};
+
+export default ProductImg;
+
+const ImgContainer = styled.div`
+  background-color: ${({ theme }) => theme.color.gray[30]};
+  border-radius: 12px;
+  overflow: hidden;
+`;
+
+const Image = styled.img`
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
```

Comment:
width, height 100%로 주고, `object-fit : cover` 주었을 때 만약 이미지의 가로세로 비율이 부모 요소의 가로세로 비율과 다르다면, `object-fit: cover` 는 이미지를 잘라내어 부모 요소에 맞출 것 같네요. 
이렇게 되면 이미지의 일부가 잘리게 될 수 있어서 contain으로 주는 게 나을 것 같아요.
[css-object-fit](https://codingeverybody.kr/css-object-fit-%EC%86%8D%EC%84%B1/)
```suggestion
  width: 100%;
  height: 100%;
  object-fit: contain;
```

Reply:
아 약간 화면이 꽉 찬게 더 이쁠것 같아서 cover로 하긴 했는데 만약 contain으로 하게 되면 배경색과 경계가 생기지 않을까요...? 배경색은 따로 지정하지 않는건가요?

Reply:
![image](https://github.com/petqua/frontend/assets/87417773/4b027558-ab30-4cbe-b57e-17a600dd8b8a)
사진이 짤리는게 더 큰 이슈이지 않을까 싶습니다.
물고기로 치면 머리가 짤릴 수도 있으니까요.
사진을 png로 받게 되면 배경색이 없어서 경계가 생기지는 않을 겁니다.